frontend/src/lib/components/PropertyFilters/PropertyFilter.js

```diff
@@ -190,8 +190,8 @@ function CohortPaneContents({ onComplete, setThisFilter, value, displayOperatorA
 export function PropertyFilter({ index, onComplete, logic }) {
     const { eventProperties, personProperties, filters } = useValues(logic)
     const { setFilter } = useActions(logic)
-    const [activeKey, setActiveKey] = useState(type === 'cohort' ? 'cohort' : 'property')
     let { key, value, operator, type } = filters[index]
+    const [activeKey, setActiveKey] = useState(type === 'cohort' ? 'cohort' : 'property')
```

Comment:
What's the difference?

Reply:
Previously this `useState` was called before `type` was defined. JS didn't complain but this caused activeKey to always be 'property' by default, even if filter type was 'cohort'.

Reply:
doh yeah that's pretty obvious. Looks good!